scripts/install.sh

```diff
@@ -68,43 +68,84 @@ trap install_success EXIT
 # Everything from this point onwards is optional.
 
 configure_systemd() {
-    if ! id ollama >/dev/null 2>&1; then
-        status "Creating ollama user..."
-        $SUDO useradd -r -s /bin/false -m -d /usr/share/ollama ollama
-    fi
-
     status "Creating ollama systemd service..."
-    cat <<EOF | $SUDO tee /etc/systemd/system/ollama.service >/dev/null
+    HOME=$(eval echo ~$(whoami))
```

Comment:
Would `HOME` always bet set?

Reply:
It is for all bash sessions, this was overkill (it covered service invocations and cron jobs), switched back to using whatever is in `$HOME`